README.md

```diff
@@ -13,6 +13,7 @@ This repo can be installed on either the same VM as the main LORIS codebase, or
 On <u>Ubuntu</u>, DICOM toolkit will be installed by the imaging install script (step 4 below). This script will _apt-get install dcmtk_.   
```

Comment:
@christinerogers 
this is minor, but can we swap steps 4 and 3 in the system requirements? I think step 3 should come before step 4.